### PR TITLE
Feat: update profile account info sub type field

### PR DESCRIPTION
**Pull request by nguyentoanit**

### CODE REVIEW

**Comment by nguyentoanit** [comment]
@moltar : I tried to update account info subtype but they still don't return.

**Comment by nguyentoanit** [comment]
@moltar : I've resolved the conflict. Please help me review them.

File: src/operations/profiles/types.ts [hunk 1]
```diff
@@ -62,6 +71,11 @@ export const AccountInfo = t.intersection([
      * The string identifier for the account name.
      */
     name: t.string,
+
+    /**
+     * The account subtype.
+     */
+    subType: AccountInfoSubType,
```
Comment (moltar) [inline src/operations/profiles/types.ts:78]: I think it's an optional field, judging by the incoming data. Will this satisfy it?

**Review by moltar — CHANGES_REQUESTED** [review]

File: src/operations/profiles/types.ts (hunk 1 above)
Comment (nguyentoanit) [reply src/operations/profiles/types.ts:78]: yeah. It's fine. At the moment, i'm also setting it is an optional field.

**Review by moltar — APPROVED** [review]

**Comment by moltar** [comment]
@nguyentoanit when can we merge this?

**Pull request merged**